### DISCUSSION TITLE
The Github Desktop App Can Not Find My Project Repository

### DISCUSSION THREAD

**Discussion by hayeonlee12** [community/community#56215] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, this is my first time to be in Github Community Forum.

I am having an issue with Github Desktop App.. I was doing my SpringBoot project but it suddenly does not recognize the repository location. There were none of this kind of issue during the project. I also deleted the Desktop App and re-installed it but it does not solves anything. 

Currently, I just took out from the Github folder and located in another folder cuz of the issue with project contents. However, I have to give the file to my team which it is possible, but it would have troubleshooting if files are overwrited in same location. 

Can I get some support for this? It appears projects are fine in the Github website, but it does not work in the Desktop App.

**Comment by DinethDilhara** [comment]
GitHub Desktop tracks your project using the hidden `.git` folder inside your project. This folder stores all the information about the repository, including history, branches, and the link to GitHub.

If you move or rename the project folder, GitHub Desktop can’t find the .git folder automatically, so you need to add the repository again in the new location.

**To fix this:**
1. Open **GitHub Desktop**.  
2. Go to **File > Add Local Repository**.  
<img width="472" height="142" alt="Screenshot 2025-09-06 at 8 38 52 PM" src="https://github.com/user-attachments/assets/aefd2543-8afc-4664-9def-5e553a4abff8" />

3. Select the folder where your Spring Boot project is now located.  
4. GitHub Desktop will recognize it again and link it with your GitHub repository.  

**If that doesn’t work:**
- Clone the repo again from GitHub into a new folder.
- Copy **only your changed files** (not the old `.git` folder) into the cloned folder.
- Use GitHub Desktop or Git commands to **commit and push** your changes.

This keeps your project connected to GitHub and safe from overwriting issues.
